etc/conda_build_matrix.py

```diff
@@ -43,11 +43,17 @@ def mkargs(py_version, npy_version, output=False):
     help='Upload the last built packages without rebuilding.',
 )
 @click.option(
+    '--allow-partial-uploads',
+    is_flag=True,
+    default=False,
+    help='Allow uploads',
```

Comment:
this should work without upload only. I will fix this because it should be more descriptive